pkg/util/testing/wrappers.go

```diff
@@ -277,8 +282,8 @@ func (w *WorkloadWrapper) appendOwnerReference(gvk schema.GroupVersionKind, name
 	return w
 }
 
-func (w *WorkloadWrapper) Annotations(kv map[string]string) *WorkloadWrapper {
-	w.ObjectMeta.Annotations = kv
+func (w *WorkloadWrapper) SetAnnotations(annotations map[string]string) *WorkloadWrapper {
```

Comment:
nit, in golang we usually don't prefix setters with `Set` so I would suggest to revert the rename.